setup.py

```diff
@@ -1,15 +1,30 @@
 # -*- coding: utf-8 -*-
 """Setup module."""
 try:
-    from setuptools import setup
+    from setuptools import setup, find_packages
 except ImportError:
     from distutils.core import setup
 
 
-def get_requires():
-    """Read requirements.txt."""
+STREAMING_REQUIREMENTS = [
+    "uvicorn",
+    "fastapi",
+    "requests"
+]
+
+
+def get_extra_ml_streaming_requires():
+    """Extract ML streaming extra requirments."""
+    requirements = open("requirements.txt", "r").read()
+    all_reqs = list(filter(lambda x: x != "", requirements.split()))
+    return [req for req in all_reqs if any(streaming in req for streaming in STREAMING_REQUIREMENTS)]
```

Comment:
It's an over-kill, I guess. Let's use `string.startswith(.)` instead.

Reply:
What do you mean by `string.startswith(.)`?

Reply:
recheck, I've rewritten this part.